handlers/jobs.go

```diff
@@ -192,6 +192,12 @@ func CreateK8sJob(inputURL string, jobConfig JobConfig) (*JobInfo, error) {
 		jobConfig.RequestMem = "0.1Gi"
 	}
 
+	quayImageIf, err := GetValueFromJSON([]byte(os.Getenv("JOB_IMAGES")), []string{jobConfig.Image})
+	if err != nil {
+		return nil, err
+	}
+	quayImage := quayImageIf.(string)
+
```

Comment:
Can `quayImage` fall back to `jobConfig.Image` if it failes to read the `JOB_IMAGES` json?

Reply:
`jobConfig.Image` is a key in `JOB_IMAGES` json not quay image so it does not make sense to do that.